src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java

```diff
@@ -1,290 +1,279 @@
+				CONFIG_DISPLAY_MQ_USER_NAME);
+
+		config.define(CONFIG_NAME_MQ_PASSWORD, Type.PASSWORD, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_PASSWORD, CONFIG_GROUP_MQ, 8, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_PASSWORD);
+
+		config.define(CONFIG_NAME_MQ_RECORD_BUILDER, Type.STRING, ConfigDef.NO_DEFAULT_VALUE, Importance.HIGH,
+				CONFIG_DOCUMENTATION_MQ_RECORD_BUILDER, CONFIG_GROUP_MQ, 9, Width.LONG,
+				CONFIG_DISPLAY_MQ_RECORD_BUILDER);
+
+		config.define(CONFIG_NAME_MQ_MESSAGE_BODY_JMS, Type.BOOLEAN, Boolean.FALSE, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_MESSAGE_BODY_JMS, CONFIG_GROUP_MQ, 10, Width.SHORT,
+				CONFIG_DISPLAY_MQ_MESSAGE_BODY_JMS);
+
+		config.define(CONFIG_NAME_MQ_RECORD_BUILDER_KEY_HEADER, Type.STRING, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_RECORD_BUILDER_KEY_HEADER, CONFIG_GROUP_MQ, 11, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_RECORD_BUILDER_KEY_HEADER);
+
+		config.define(CONFIG_NAME_MQ_SSL_CIPHER_SUITE, Type.STRING, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_SSL_CIPHER_SUITE, CONFIG_GROUP_MQ, 12, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_SSL_CIPHER_SUITE);
+
+		config.define(CONFIG_NAME_MQ_SSL_PEER_NAME, Type.STRING, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_SSL_PEER_NAME, CONFIG_GROUP_MQ, 13, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_SSL_PEER_NAME);
+
+		config.define(CONFIG_NAME_MQ_SSL_KEYSTORE_LOCATION, Type.STRING, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_SSL_KEYSTORE_LOCATION, CONFIG_GROUP_MQ, 14, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_SSL_KEYSTORE_LOCATION);
+
+		config.define(CONFIG_NAME_MQ_SSL_KEYSTORE_PASSWORD, Type.PASSWORD, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_SSL_KEYSTORE_PASSWORD, CONFIG_GROUP_MQ, 15, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_SSL_KEYSTORE_PASSWORD);
+
+		config.define(CONFIG_NAME_MQ_SSL_TRUSTSTORE_LOCATION, Type.STRING, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_SSL_TRUSTSTORE_LOCATION, CONFIG_GROUP_MQ, 16, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_SSL_TRUSTSTORE_LOCATION);
+
+		config.define(CONFIG_NAME_MQ_SSL_TRUSTSTORE_PASSWORD, Type.PASSWORD, null, Importance.MEDIUM,
+				CONFIG_DOCUMENTATION_MQ_SSL_TRUSTSTORE_PASSWORD, CONFIG_GROUP_MQ, 17, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_SSL_TRUSTSTORE_PASSWORD);
+
+		config.define(CONFIG_NAME_MQ_BATCH_SIZE, Type.INT, CONFIG_VALUE_MQ_BATCH_SIZE_DEFAULT,
+				ConfigDef.Range.atLeast(CONFIG_VALUE_MQ_BATCH_SIZE_MINIMUM), Importance.LOW,
+				CONFIG_DOCUMENTATION_MQ_BATCH_SIZE, CONFIG_GROUP_MQ, 18, Width.MEDIUM,
+				CONFIG_DISPLAY_MQ_BATCH_SIZE);
+
+		config.define(CONFIG_NAME_MQ_MESSAGE_MQMD_READ, Type.BOOLEAN, Boolean.FALSE, Importance.LOW,
+				CONFIG_DOCUMENTATION_MQ_MESSAGE_MQMD_READ, CONFIG_GROUP_MQ, 19, Width.SHORT,
+				CONFIG_DISPLAY_MQ_MESSAGE_MQMD_READ);
+
+		config.define(CONFIG_NAME_MQ_JMS_PROPERTY_COPY_TO_KAFKA_HEADER, Type.BOOLEAN, CONFIG_VALUE_MQ_JMS_PROPERTY_COPY_TO_KAFKA_HEADER_DEFAULT, Importance.LOW,
```

Comment:
Please use the same config.define method signature as the existing options.

Reply:
@AndrewJSchofield Changed as per the suggestion.